pombola/feedback/views.py

```diff
@@ -14,19 +16,24 @@ def add(request):
 
     submit_was_success = False
     return_to_url      = None
+    submit_error_message  = None
 
     # If it is a post request try to create the feedback
     if request.method == 'POST':
         form = FeedbackForm( request.POST )
+        recaptcha_response = request.POST.get("g-recaptcha-response", None)
```

Comment:
Just nit-picking, but `.get("g-recaptcha-response", None)` is the same as `.get("g-recaptcha-response")`. `.get` defaults to `None` :)

Reply:
Today I learned! :)